subcommands/keys/ca_show.go

```diff
@@ -74,22 +77,18 @@ func doShowCA(cmd *cobra.Command, args []string) {
 	}
 
 	fmt.Println("## Factory root certificate")
-	if prettyFormat {
-		prettyPrint(resp.RootCrt)
-	} else {
-		fmt.Println(resp.RootCrt)
-	}
+	printOneCert(resp.RootCrt)
 	fmt.Println("## Server TLS Certificate")
-	if prettyFormat {
-		prettyPrint(resp.TlsCrt)
-	} else {
-		fmt.Println(resp.TlsCrt)
-	}
+	printOneCert(resp.TlsCrt)
 	fmt.Println("\n## Device Authentication Certificate(s)")
+	printOneCert(resp.CaCrt)
```

Comment:
Is this going to print all device CAs or just one?

Reply:
It prints one cert file; which may contain several certificates.

I know the name might look misleading after that my statement.
Maybe, `printCerts` sounds better?